app/config.go

```diff
@@ -18,14 +17,20 @@ type Config struct {
 	Type        string             `json:"type"`
 	Version     string             `json:"version"`
 	Description string             `json:"description"`
-	Properties  interface{}        `json:"properties"`
+	Properties  []AppProperty      `json:"properties"`
 	Triggers    []*trigger.Config  `json:"triggers"`
 	Resources   []*resource.Config `json:"resources"`
 
 	//for backwards compatibility
 	Actions []*action.Config `json:"actions"`
 }
 
+type AppProperty struct {
```

Comment:
Why not use Attribute?  It has the same exact members

Reply:
@fm-tibco : Make sense. Check updated PR.